docs/book/user-guide/tutorials/managing-scheduled-pipelines.md

```diff
@@ -0,0 +1,303 @@
+┃            0abcdef1            │ 10_00_00_000000               │        │                 │                     │          │                 │         │                     ┃
+┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┷━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┷━━━━━━━━┷━━━━━━━━━━━━━━━━━┷━━━━━━━━━━━━━━━━━━━━━┷━━━━━━━━━━┷━━━━━━━━━━━━━━━━━┷━━━━━━━━━┷━━━━━━━━━━━━━━━━━━━━━┛
+Page `(1/1)`, `2` items found for the applied filters.
+```
+
+## Step 3.1: Verify Schedule on GCP
+
+To ensure the schedule was properly created in Vertex AI, we can verify it using the Google Cloud SDK:
+
+```python
+from google.cloud import aiplatform
+
+# List all Vertex schedules
+vertex_schedules = aiplatform.PipelineJobSchedule.list(
+    filter=f'display_name="{schedule.name}"',
+    location="us-central1"  # Replace with your Vertex AI region
+)
+
+our_vertex_schedule = next(
+    (s for s in vertex_schedules if s.display_name == schedule_name), None
+)
+
+if our_vertex_schedule:
+    print(
+        f"Vertex AI schedule '{our_vertex_schedule.display_name}' created successfully!"
+    )
+    print(f"State: {our_vertex_schedule.state}")
+    print(f"Cron expression: {our_vertex_schedule.cron}")
+    print(
+        f"Max concurrent run count: {our_vertex_schedule.max_concurrent_run_count}"
+    )
+else:
+    print("Schedule not found in Vertex AI!")
+```
+
+{% hint style="warning" %}
+Make sure to replace `us-central1` with your actual Vertex AI region. You can find your region in the Vertex AI settings or by checking the `location` parameter in your Vertex orchestrator configuration.
+{% endhint %}
+
+## Step 4: Update the Schedule
+
+Sometimes we need to modify an existing schedule. Since ZenML doesn't support direct schedule updates, we'll need to delete the old schedule and create a new one.
+
+```python
+# First, delete the existing schedule
+client.delete_schedule("daily-data-processing")
+
+# Create a new schedule with updated parameters
+new_schedule = Schedule(
+    name="daily-data-processing",
+    cron_expression="0 10 * * *"  # Changed to 10 AM
+)
+
+# Attach the new schedule to our pipeline
+updated_pipeline = daily_data_pipeline.with_options(schedule=new_schedule)
+
+# Run the pipeline to create the new schedule
+updated_pipeline()
+```
+
+Using the CLI to delete a schedule:
+
+```bash
+# Delete a specific schedule
+zenml pipeline schedule delete daily-data-processing
+
+# rerun the pipeline to create the new schedule
+python run.py # or whatever you named your script
+```
+
+> **Important**: When updating schedules, you should also delete the
+> corresponding schedule in your orchestrator (Vertex AI in this example). You
+> can do this through the Google Cloud Console or using the orchestrator's API
+> (see below for code example).
+> ZenML's delete command may not always completely remove the underlying
+> orchestrator schedule.
+
+## Step 4.1: Delete Schedule on GCP
```

Comment:
This should be linked somehow to Step 6 too right? it feels a bit confusing now to have deletion here and then not in the Clean up

Reply:
So just have one place which talks about deleting schedules and the others just
link to it?

Reply:
@strickvl yeah i think so?